docs/book/src/developer/tilt.md

```diff
@@ -70,7 +70,7 @@ documentation](https://docs.tilt.dev/api.html#api.default_registry) for more det
 for more details.
 
 **kustomize_substitutions** (Map{String: String}, default={}): An optional map of substitutions for `${}`-style placeholders in the
-provider's yaml. **Note**: It's recommended to enable the following feature flags for local dev environment to ensure e2e tests run through:
+provider's yaml. **Note**: It's recommended to enable the following feature flags for local dev environment. This will enable all e2e tests to run through:
```

Comment:
what about
```suggestion
provider's yaml. **Note**: It's recommended to enable the following feature flags for local dev environment. When running E2E tests locally using an exisiting cluster managed by Tilt, this will enable all e2e tests to run through:
```

Reply:
Fixed in a similar way